src/jdk.jartool/share/classes/sun/tools/jar/GNUStyleOptions.java

```diff
@@ -326,11 +340,17 @@ static int parseOptions(Main jartool, String[] args) throws BadArgs {
 
         // process options
         for (; count < args.length; count++) {
-            if (args[count].charAt(0) != '-' || args[count].equals("-C") ||
-                args[count].equals("--release"))
+            String name = args[count];
+            if (name.charAt(0) != '-' || name.equals("-C"))
                 break;
+            if (name.equals("--release")) {
+                // JDK-8316804: stay here when in `--describe-module`
```

Comment:
It's my understanding that we don't include the bug ids in the code comments within the JDK. The rest of the comment without the bug id I think would be sufficient.